Ginger/Ginger/UserControlsLib/TextEditor/ValueExpression/ValueExpressionMockDataEditorPage.xaml.cs

```diff
@@ -0,0 +1,325 @@
+﻿#region License
+/*
+Copyright © 2014-2024 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using Amdocs.Ginger.Common;
+using Bogus;
+using Ginger.UserControlsLib.TextEditor.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Controls;
+
+namespace Ginger.UserControlsLib.TextEditor.ValueExpression
+{
+    /// <summary>
+    /// Interaction logic for ValueExpressionMockDataEditorPage.xaml
+    /// </summary>
+    public partial class ValueExpressionMockDataEditorPage : Page
+    {
+        SelectedContentArgs mSelectedContentArgs;
+        Context mContext;
+        string mObj;
+        public ValueExpressionMockDataEditorPage(Context context, SelectedContentArgs SelectedContentArgs, string obj, string function, string Locale, string MockExpression)
+        {
+            InitializeComponent();
+            mContext = context;
+            mSelectedContentArgs = SelectedContentArgs;
+            mObj = obj;
+            int datasetStartindex = MockExpression.IndexOf('=');
```

Comment:
Add error handling in case of invalid input values

Reply:
Done